flask_monitoringdashboard/views/reporting.py

```diff
@@ -19,7 +19,7 @@
 
 
 def get_date(p):
-    return datetime.utcfromtimestamp(int(request.args.get(p)))
+    return datetime.fromtimestamp(int(request.args.get(p)),tz = datetime.datetime.now(datetime.timezone.utc))
```

Comment:
one small change: the `tz=...` expects a timezone object not a datetime. can you fix this?

Reply:
ok. no worries; i fixed it myself.